tests/test_remote_settings.py

```diff
@@ -13,9 +14,10 @@ def test_build_remote_setting_poll_url():
     res = RemoteSettings(project_id=403427,
                          host="https://notifier-configs.airbrake.io",
                          config=config)
+    print(res._poll_url(res._data))
```

Comment:
We don't want to print anything to the console when we run tests

Reply:
My bad, I'll remove print.